lib/vsc/install/commontest.py

```diff
@@ -273,3 +275,23 @@ def test_prospector(self):
 
         failures = run_prospector(self.setup.REPO_BASE_DIR)
         self.assertFalse(failures, "prospector failures: %s" % pprint.pformat(failures))
+
+    def test_jenkinsfile(self):
+        """Test whether Jenkinsfile is in place, and was auto-generated with correct revision."""
+
+        self.assertTrue(os.path.exists('Jenkinsfile'))
+        with open('Jenkinsfile') as fh:
+            txt = fh.read()
+
+        regex = re.compile(r'^// \[revision: %s\]$' % re.escape(JENKINSFILE_REVISION), re.M)
```

Comment:
this is pointless imho. just generate the expected text and compare the generated text with text from file (it gives you a nice diff too).

Reply:
removed